wandb/sdk/launch/sweeps/scheduler_sweep.py

```diff
@@ -46,7 +46,7 @@ def __init__(
         # (emulating a real agent) and add new runs to the launch queue. The
         # launch agent is the one that actually runs the training workloads.
         self._workers: Dict[int, _Worker] = {}
-        self._num_workers: int = num_workers
+        self._num_workers: int = int(num_workers)
```

Comment:
is casting here safe?

Reply:
So long as the sweep shema.json checks it, it shouldn't be necessary?